src/store/storeConfig.ts

```diff
@@ -284,8 +288,13 @@ export const storeConfig: StoreOptions<State> = {
     },
 
     // update the selected mesh
-    updateSelectedMesh({ commit }, mesh) {
-      ClientStorage.set('selectedMesh', mesh)
+    updateSelectedMesh({ commit }, mesh: string | null) {
+      if (mesh !== null) {
+        ClientStorage.set('selectedMesh', mesh)
+      } else {
+        ClientStorage.remove('selectedMesh')
+      }
+
```

Comment:
Did this happen in another PR also?

Reply:
Oh yeah, true, I must’ve applied that to both PRs while I was looking at it. I’m removing it.